interface/code_systems/list_staged.php

```diff
@@ -205,9 +205,20 @@
                     array_push($revisions, $temp_date);
                     $supported_file = 1;
                 } elseif (preg_match("/SnomedCT_SpanishRelease-es_PRODUCTION_([0-9]{8})[0-9a-zA-Z]{8}.zip/", $file, $matches)) {
-                    // Hard code the version SNOMED feed to be International:Spanish
+                    // Hard code the version SNOMED feed to be International:SpanishLang
                     //
-                    $version = "International:Spanish";
+                    $version = "International:SpanishLang";
+                    $rf2 = true;
+                    $date_release = substr($matches[1], 0, 4) . "-" . substr($matches[1], 4, -2) . "-" . substr($matches[1], 6);
+                    $temp_date = array('date' => $date_release, 'version' => $version, 'path' => $mainPATH . "/" . $matches[0]);
+                    array_push($revisions, $temp_date);
+                    $supported_file = 1;
+                } elseif (preg_match("/SnomedCT_Argentina-EditionRelease_PRODUCTION_([0-9]{8})[0-9a-zA-Z]{8}.zip/", $file, $matches)) {
+                    // Hard code the version SNOMED feed to be International:SpanishLang. Argentinian format.
+                    // Contains the Snapshot, Delta and Full folders with the Argentina extension files already unified
+                    // with the Spanish and International files
+                    //
+                    $version = "International:SpanishLang";
```

Comment:
@stephenwaite If I'm understanding correctly your saying we should put the version to be something like "International:SpanishLang:Argentina"?

Reply:
thinking it should be `International:Argentina-Edition`

I think what Brady was getting at is to make sure the revision and updating can be distinct

Reply:
@luisuriarte did you want to address this change? I know its been a while but I'd like to either get this PR in or close it.